spec/features/welcome/login_spec.rb

```diff
@@ -1,11 +1,10 @@
 RSpec.describe 'user can log in' do
   describe 'using google_oauth2', type: :feature do
-
-    it 'user authenticates via Google, and is redirected to their dashboard where they see Welcome, user_email' do
+    it "user authenticates via Google, and is redirected to their dashboard where they see 'Welcome, <user_email>'" do
```

Comment:
@rrabinovitch good call on moving the log-in spec to it's own test file in the welcome folder. Makes more sense for sure!